src/libs/antares/study/parameters/adq-patch-params.cpp

```diff
@@ -128,8 +136,6 @@ void CurtailmentSharing::addProperties(IniFile::Section* section) const
 // ------------------------
 void AdqPatchParams::reset()
 {
-    enabled = false;
```

Comment:
This `enabled` var is related to adq patch, not the same as the one in local matching
This change is also present on develop, I'll do a PR to fix it

Reply:
https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2420

Reply:
done